test/Microsoft.AspNet.Razor.Test/TagHelpers/TagHelperParseTreeRewriterTest.cs

```diff
@@ -19,6 +19,282 @@ namespace Microsoft.AspNet.Razor.Test.TagHelpers
 {
     public class TagHelperParseTreeRewriterTest : CsHtmlMarkupParserTestBase
     {
+        public static TheoryData PrefixedTagHelperBoundData
+        {
+            get
+            {
+                var factory = CreateDefaultSpanFactory();
+                var blockFactory = new BlockFactory(factory);
+                var availableDescriptors1 = new TagHelperDescriptor[]
+                {
+                    new TagHelperDescriptor(
+                        prefix: "th:",
+                        tagName: "myth",
+                        typeName: "mythTagHelper",
+                        assemblyName: "SomeAssembly",
+                        attributes: Enumerable.Empty<TagHelperAttributeDescriptor>()),
+                    new TagHelperDescriptor(
+                        prefix: "th:",
+                        tagName: "myth2",
+                        typeName: "mythTagHelper2",
+                        assemblyName: "SomeAssembly",
+                        attributes: new []
+                        {
+                            new TagHelperAttributeDescriptor(
+                                name: "bound",
+                                propertyName: "Bound",
+                                typeName: typeof(bool).FullName),
+                        })
+                };
+                var availableDescriptors2 = new TagHelperDescriptor[]
+                {
+                    new TagHelperDescriptor(
+                        prefix: "PREFIX",
+                        tagName: "myth",
+                        typeName: "mythTagHelper",
+                        assemblyName: "SomeAssembly",
+                        attributes: Enumerable.Empty<TagHelperAttributeDescriptor>()),
+                    new TagHelperDescriptor(
+                        prefix: "PREFIX",
+                        tagName: "myth2",
+                        typeName: "mythTagHelper2",
+                        assemblyName: "SomeAssembly",
+                        attributes: new []
+                        {
+                            new TagHelperAttributeDescriptor(
+                                name: "bound",
+                                propertyName: "Bound",
+                                typeName: typeof(bool).FullName),
+                        })
+                };
+                var availableDescriptors3 = new TagHelperDescriptor[]
+                {
+                    new TagHelperDescriptor(
+                        prefix: "myth",
+                        tagName: "myth",
```

Comment:
I wasn't specific enough in my `<myth/>` suggestion.  meant this should be a catch-all descriptor.  covers the case where Razor source uses a tag name that exactly matches the prefix of a catch-all descriptor.